repository/src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -47,6 +47,6 @@
     <include file="db/changelog/db.changelog-rdm-114.xml"/>
     <include file="db/changelog/db.changelog-rdm-2917.xml"/>
     <include file="db/changelog/db.changelog-rdm-2917-cleanup.xml"/>
-
+    <include file="db/changelog/db.changelog-rdm-1202.xml"/>
```

Comment:
changes to DB are part of separate PR #190

Reply:
why do they show up here as well?

Reply:
I had to pull the DB branch into this one for successful build (due to changes to Entity), local testing etc.